packages/flutter/lib/src/material/input.dart

```diff
@@ -37,11 +36,8 @@ class Input extends StatefulComponent {
     assert(key != null);
   }
 
-  /// The initial editable text for the input field.
-  final String initialValue;
-
-  /// The initial selection for this input field.
-  final TextSelection initialSelection;
+  /// The editable text for the input field.
```

Comment:
"editable text" is an odd description of an immutable field whose type is an immutable object...